src/style.css

```diff
@@ -0,0 +1,120 @@
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 14px;
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 45px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  font-size: 11px;
+  margin-top: 5px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 15px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star,
+.stars_5 .card__star {
+  width: 15px;
+  height: 15px;
+
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.card__star:nth-child(1n+2) {
+  margin-left: 4px;
+}
+
+.stars_0 .card__star,
+.stars_1 .card__star:nth-child(1n+2),
+.stars_2 .card__star:nth-child(1n+3),
+.stars_3 .card__star:nth-child(1n+4),
+.stars_4 .card__star:nth-child(1n+5) {
+  background: url("images/star.svg") no-repeat center;
+}
+
+.card__feedbacks {
+  font-size: 11px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 20px;
+}
+
+.card__prop {
+  font-size: 11px;
+  line-height: 13px;
+
+  margin-bottom: 1px;
+}
+
+.card__value {
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.card__btn {
+  padding: 12px 0;
+  margin-top: 20px;
+
+  font-size: 14px;
+  color: white;
+  font-weight: bold;
+  text-decoration: none;
+  line-height: 16px;
+  text-align: center;
+
+  background: #00acdc;
+  border-radius: 5px;
+}
+
+.card__btn:hover {
+  padding: 11px 0;
+
+  color: #00acdc;
+
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
don't add border to the button on hover. Add it to initial button state and make transparent
Checklist

Reply:
removed border from the hover effect